### ISSUE TITLE
[Experiment] LR pilot study — short experiments to find optimal learning rate

### ISSUE THREAD

**Issue by fritzprix** [fritzprix/systemone-lite#12]
# Issue: LR pilot study — short experiments to find optimal learning rate

## Background

Run `spatial-v2-s1-cloze` (52,124 steps) ended with 54.25% val accuracy, but the 40k checkpoint hit 56.00% (unreproducible). The resume caused a 190× LR jump (7.78e-8 → 1.48e-5), making results inconclusive.

The postmortem recommends a short pilot to find optimal LR before full training.

### Trials

| Trial | Peak LR | Effective Batch | Length | Notes |
|---|---:|---:|---|---|
| A | `2e-6` | 16 (4 phys × 4 accum) | 5–10k | Conservative |
| B | `5e-6` | 16 | 5–10k | Medium |
| C | `1e-5` | 16 | 5–10k | Aggressive |

### Shared Settings

- Physical batch: 4
- Gradient accumulation: 4
- Warmup: 100–500 optimizer steps
- Scheduler: cosine or constant-with-warmup
- Same checkpoint, same data order, same seed (7) — **only LR varies**
- Eval every 2.5k steps
- Winner validated with 3 seeds

### Starting Point

Use the **pre-resume checkpoint** (step ~20k, LR 7.78e-8) or the 40k checkpoint (56% val) as starting point.

### Step 1: Run Trials A, B, C

```bash
# Trial A
python scripts/chess_finetune.py \
  --resume checkpoints/spatial-v2-s1-cloze-step-20000.pt \
  --lr 2e-6 \
  --peak-lr 2e-6 \
  --batch-size 4 \
  --grad-accum 4 \
  --max-steps 10000 \
  --warmup-steps 500 \
  --scheduler cosine \
  --seed 7 \
  --wandb-project systemone-lite \
  --wandb-run trial-A-lr-2e6

# Trial B
python scripts/chess_finetune.py \
  --resume checkpoints/spatial-v2-s1-cloze-step-20000.pt \
  --lr 5e-6 \
  --peak-lr 5e-6 \
  --batch-size 4 \
  --grad-accum 4 \
  --max-steps 10000 \
  --warmup-steps 500 \
  --scheduler cosine \
  --seed 7 \
  --wandb-project systemone-lite \
  --wandb-run trial-B-lr-5e6

# Trial C
python scripts/chess_finetune.py \
  --resume checkpoints/spatial-v2-s1-cloze-step-20000.pt \
  --lr 1e-5 \
  --peak-lr 1e-5 \
  --batch-size 4 \
  --grad-accum 4 \
  --max-steps 10000 \
  --warmup-steps 500 \
  --scheduler cosine \
  --seed 7 \
  --wandb-project systemone-lite \
  --wandb-run trial-C-lr-1e5
```

### Step 2: Analyze results

- Compare val accuracy curves
- Compare training loss curves (raw + rolling mean)
- Select winner (highest sustained val accuracy, not single spike)

### Step 3: Validate winner with 3 seeds

Run the winning LR with seeds 7, 42, 137. Report mean ± std.

## Success Criteria

- [ ] One trial shows sustained val accuracy > 55% (not single checkpoint)
- [ ] Winner validated across 3 seeds with low variance (std < 1.5%)
- [ ] Learning rate curve is smooth (no discontinuities)
- [ ] Training loss decreases monotonically after warmup (or plateaus at reasonable level)

## Notes

- If 40k checkpoint is available, evaluate it on a **separate holdout set** but **do not** adopt it as the final model yet
- Each trial should be a separate W&B run (not resuming into the same run)
- Track all metrics from Issue #10 (raw loss, EMA, gradient norm, etc.)
